lib/hexpm/repository/package.ex

```diff
@@ -383,6 +383,16 @@ defmodule Hexpm.Repository.Package do
     )
   end
 
+  defp sort(query, :total_dependants) do
+    from(
+      p in query,
+      join: d in PackageDependant,
+      on: p.name == d.name,
+      group_by: [p.name, p.id],
+      order_by: [desc: count(p.name)]
+    )
```

Comment:
Do you think the query is easier to understand like this?

```elixir
from(
  p in query,
  join: d in PackageDependant,
  on: p.name == d.name,
  group_by: [p.id],
  order_by: [desc: count(d.dependant_id)]
)

Reply:
Yes, I think that works better. I've updated the code.